org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterIas.java

```diff
@@ -115,21 +122,38 @@ protected boolean hasIasZoneInputCluster(ZigBeeEndpoint endpoint) {
 
     @Override
     public void commandReceived(ZclCommand command) {
+        logger.debug("{}: ZigBee command report {}", endpoint.getIeeeAddress(), command);
         if (command instanceof ZoneStatusChangeNotificationCommand) {
             ZoneStatusChangeNotificationCommand zoneStatus = (ZoneStatusChangeNotificationCommand) command;
-            switch (channel.getAcceptedItemType()) {
-                case "Switch":
-                    updateChannelState(((zoneStatus.getZoneStatus() & bitTest) != 0) ? OnOffType.ON : OnOffType.OFF);
-                    break;
-                case "Contact":
-                    updateChannelState(((zoneStatus.getZoneStatus() & bitTest) != 0) ? OpenClosedType.OPEN
-                            : OpenClosedType.CLOSED);
-                    break;
-                default:
-                    logger.warn("{}: Unsupported item type {}", endpoint.getIeeeAddress(),
-                            channel.getAcceptedItemType());
-                    break;
+            updateChannelState(zoneStatus.getZoneStatus());
+        }
+    }
+
+    @Override
+    public void attributeUpdated(ZclAttribute attribute) {
+        logger.debug("{}: ZigBee attribute reports {}", endpoint.getIeeeAddress(), attribute);
+        if (attribute.getCluster() == ZclClusterType.OCCUPANCY_SENSING
```

Comment:
One question, Chris: Why should this code relating to occupancy sensing go into the abstract base class for all IAS converters. Isn't this something that should only go into the subclasses that deal with occupancy sensing?